rholang/reference_doc/data_structures/README.md

```diff
@@ -114,6 +114,7 @@ Converts string to UTF-8 byte array. Returns an error when called with arguments
 #### Ordered data structure methods
 - `length()`: the number of elements in a byte array, list, or string
 - `slice(m,n)`: slice of a byte array, list, or string
+- `toSet()`: generates a Set of list elements eliminating guplicates
```

Comment:
It's a typo in `guplicates`.

Please make it clear that method is applicable to list and map structures.